docs/cpp/faq-cpp.md

```diff
@@ -66,3 +72,61 @@ When you open a workspace for the first time, the extension adds `${workspaceRoo
 ## How do I recreate the IntelliSense database?
 
 Starting in version 0.12.3 of the extension, there is a command to reset your IntelliSense database. Open the Command Palette (`kb(workbench.action.showCommands)`) and choose the **C_Cpp: Reset IntelliSense Database** command.
+
+## What is the ipch folder?
+
+The language server caches information about included header files to improve the performance of IntelliSense. When you edit C/C++ files in your workspace folder, the language server will store cache files in the `ipch` folder. By default, the `ipch` folder is stored under the user directory. Specifically, it is stored under `%LocalAppData%\vscode-cpptools` on Windows, and for Linux and macOS it is under `~/.vscode-cpptools`. By using the user directory as the default path, it will create one cache location per user for the extension. As the cache size limit is applied to a cache location, having one cache location per user will limit the disk space usage of the cache to that one folder for everyone using the default setting value.
```

Comment:
Change `%LocalAppData%\vscode-cpptools` to `%LocalAppData%/Microsoft/vscode-cpptools\`